tests/integration/kubernetes/k8s-guest-pull-image.bats

```diff
@@ -16,8 +16,9 @@ setup() {
     [ "${SNAPSHOTTER:-}" = "nydus" ] || skip "None snapshotter was found but this test requires one"
 
     setup_common
+    get_pod_config_dir
     unencrypted_image="quay.io/prometheus/busybox:latest"
-    large_image="quay.io/confidential-containers/test-images:largeimage"
+    large_image="ghcr.io/confidential-containers/test-container:big-size"
```

Comment:
Out of interest, is there any particular reason for ghcr over quay.io?

Reply:
The reason for selecting a different image is that Image `quay.io/confidential-containers/test-images:largeimage` is too large (larger than 2GB) and may exceed the default kubelet timeout of 120s in CI. Even if we set the createContainer timeout value to more than 120 seconds on the kata side, it might still fail. Therefore, I chose a smaller image (size = 956MB) that can be pulled within 120 seconds.

Reply:
You would need to set the remote request timeout in the kubelet. This is independent of what we set in kata since the context is created in the kubelet.
Do we have the tools in the ci to update kubelet arguments?

Reply:
The 120s will also break the GPU use case..

Reply:
@zvonkok I think updating the kubelet configuration might be challenging while the test is running. And I think we can focus on the feature that allows configuring the timeout specifically for pulling larger images. If users need to pull an image with a timeout exceeding 120 seconds, they can update the configuration of kubelet offline. While I'm not an expert on this, I would like any advice from the experts if possible.